### PR TITLE
[Profile] Modify for swap status

### PR DESCRIPTION
**Pull request by jihochu**
This patch adds swap status and policies.

Deallocation memory size is added, and swap IN/OUT status and cache policy columns are added.

Signed-off-by: Jiho Chu <jiho.chu@samsung.com>

---

Example output:


```
                         event                size               total                                                        info       dur              policy      swap                                                                                                                                                                         
==========================================================================================================================================================================================================================                                                                                                                         
Initialize                                                                                                                                                                                                                                                                                                                                         
                         ALLOC                 600                 600                    CacheElem(/home/zio/weight_pool77939) #1                 ALWAYS_SYNCED        IN                                                                                                                                                                         
                       DEALLOC                 600                   0                    CacheElem(/home/zio/weight_pool77939) #1        31       ALWAYS_SYNCED       OUT                                                                                                                                                                         
                         ALLOC                  24                  24                    CacheElem(/home/zio/weight_pool77939) #2                 ALWAYS_SYNCED        IN                                                                                                                                                                         
                       DEALLOC                  24                   0                    CacheElem(/home/zio/weight_pool77939) #2        15       ALWAYS_SYNCED       OUT                                                                                                                                                                         
                         ALLOC                7200                7200                    CacheElem(/home/zio/weight_pool77939) #3                 ALWAYS_SYNCED        IN                                                                                                                                                                         
                       DEALLOC                7200                   0                    CacheElem(/home/zio/weight_pool77939) #3        54       ALWAYS_SYNCED       OUT                                                                                                                                                                         
                         ALLOC                  48                  48                    CacheElem(/home/zio/weight_pool77939) #4                 ALWAYS_SYNCED        IN                                                                                                                                                                         
                       DEALLOC                  48                   0                    CacheElem(/home/zio/weight_pool77939) #4         5       ALWAYS_SYNCED       OUT                                                                                                                                                                         
                         ALLOC                7680                7680                    CacheElem(/home/zio/weight_pool77939) #5                 ALWAYS_SYNCED        IN
```

### CODE REVIEW

**Review by DonghakPark — APPROVED** [review]
LGTM!

File: nntrainer/utils/profiler.h [hunk 1]
```diff
@@ -111,6 +134,9 @@ struct ProfileEventData {
   /* total allocation size */
   size_t alloc_total;
 
+  std::string cache_policy;
+  bool cache_swap;
```
Comment (jijoongmoon) [inline nntrainer/utils/profiler.h:138]: Could you add explanation or comment about cache_swap?

**Review by jijoongmoon — APPROVED** [review]
LGTM except for a minor comments.

**Comment by baek2sm** [comment]
LGTM

**Pull request merged**